knowledge/responses/peta_is_bad.md

```diff
@@ -1,35 +1,15 @@
 
-Here's something of a quick overview read:
+For instance, one of the most organized campaigns against PETA claims that PETA euthanizes animals like any regular shelter. <a href="https://www.sourcewatch.org/index.php/Richard_Berman_cares_about_animals:_clients_exposed"> This campaign was coordinated by the Center for Consumer Freedom </a>, an organization sponsored, among others, by meat and tobacco companies.
 
-<a href="https://www.reddit.com/r/PETA/comments/a47xuq/do_you_think_peta_is_evil_click_here/">Do you think PETA is evil? Click here!</a>
+PETA's blitz-style campaigns often draws negative publicity to their targets, and contain a good dose of shock-value. This approach, however disruptive, has yielded excelent results, year after year. For example, the iditarod dog race has lost several sponsors, several countries have banned fur farms, food suppliers and cosmetic companies quit animal testing, aquariums lost their rights to abuse dolphins and whales, use of animals in circus in several jurisdictions became illegal, etc. Even if you are not vegan, I believe you agree whales and elephants shouldn't be confined in small spaces and tortured to perform tricks. All these victories are thanks to PETA.
 
-Here's some other resources that go over what PETA actually does and why they do it, I hope you can take some time to read into it as while we may not agree with PETA's marketing and advertising, they do do a lot of good around the world. That's why Peter and other GOT actors have partnered with PETA to spread that message. Hope this helps mate.
+Jonathan Safran Foer, in his book Eating Animals, says: _"Whatever one thinks of them, no organization strikes fear in the factory farm industry and its allies more than PETA. They are effective. When PETA targeted fast-food companies, the most famous and powerful welfare scientist in the country, Temple Grandin (who has designed more than half the cattle slaughter facilities in the nation), said she saw more improvement in welfare in one year than she had seen in her entire thirty-year career previously. Arguably the biggest PETA hater on the planet, Steve Kopperud (a meat industry consultant who has given anti-PETA seminars for a decade), puts it this way: “There’s enough understanding in the industry now of what PETA’s capable of to put the fear of God into many executives.” It didn’t surprise me to learn that companies of all kinds regularly negotiate with PETA and quietly make changes in their animal welfare policies to avoid being publicly targeted by the group."_ 
```

Comment:
```suggestion
Jonathan Safran Foer, in his book "Eating Animals," says: 

> Whatever one thinks of them, no organization strikes fear in the factory farm industry and its allies more than PETA. They are effective. When PETA targeted fast-food companies, the most famous and powerful welfare scientist in the country, Temple Grandin (who has designed more than half the cattle slaughter facilities in the nation), said she saw more improvement in welfare in one year than she had seen in her entire thirty-year career previously. Arguably the biggest PETA hater on the planet, Steve Kopperud (a meat industry consultant who has given anti-PETA seminars for a decade), puts it this way: “There’s enough understanding in the industry now of what PETA’s capable of to put the fear of God into many executives.” It didn’t surprise me to learn that companies of all kinds regularly negotiate with PETA and quietly make changes in their animal welfare policies to avoid being publicly targeted by the group.
```